compose/cli/main.py

```diff
@@ -377,9 +377,20 @@ def down(self, options):
             -t, --timeout TIMEOUT   Specify a shutdown timeout in seconds.
                                     (default: 10)
         """
+        environment = Environment.from_env_file(self.project_dir)
+        ignore_orphans = environment.get_boolean('COMPOSE_IGNORE_ORPHANS')
+
+        if ignore_orphans and options['--remove-orphans']:
+            log.info("COMPOSE_IGNORE_ORPHANS is set, --remove-orphans flag is being ignored.")
```

Comment:
Maybe `log.warn`? You could even argue for erroring out here, since the two options are conceptually incompatible.

Reply:
True, I guess accidentally specifying both here could cause some unexpected behavior if it 'fails' silently. Will change it.